src/main/java/org/breedinginsight/api/auth/AuthServiceLoginHandler.java

```diff
@@ -43,6 +50,7 @@
 
     @Inject
     private UserService userService;
+    private String loginSuccessUrlCookieName = "redirect-login";
```

Comment:
Do we want the cookie name in the config file? You could then inject it here.

Reply:
changed and pushed